File: config/config-sbc2ha/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/sensor/oneWire/therm/fs/DS18B20busFs.java
package com.dfi.sbc2ha.config.sbc2ha.definition.sensor.oneWire.therm.fs;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.PlatformType;
import com.dfi.sbc2ha.config.sbc2ha.definition.sensor.oneWire.therm.DS18B20;

public class DS18B20busFs extends DS18B20 {


    public DS18B20busFs() {
        super();
        platform = PlatformType.DALLAS;
    }
}

File: config/editor/src/app/definition/sensor/oneWire/therm/fs/DS18B20busFs.ts
import {DS18B20} from "../DS18B20";
import {PlatformType} from "../../../../enums/PlatformType";

export class DS18B20busFs extends DS18B20 {

    constructor(name: string, busId: string, address: string) {
        super(PlatformType.DALLAS, name, busId, address);
    }
}
